ivre/analyser.py

```diff
@@ -327,6 +347,34 @@ def count(self):
             count['anomalies'] = len(self._anomalies)
         return count
 
+    def set_labels(self, session=None):
+        """
+        Tag hosts in database with their respective cluster number
+        and anomaly status.
+        `session` must be None or a positive integer representing
+        the group tag in database. The possible group tags are
+        'analyse', 'analyse_1', 'analyse_2', ...
+        """
+        session_name = 'analyse'
+        if session and int(session) > 0:
+            session_name += '_' + str(int(session))
```

Comment:
When possible, prefer string formatting (`session_name += '_%d' + int(session)`).

Here, you can even drop the `int(session) > 0` test (since the docstring is very clear about what is expected here) and write:
```Python
session_name = 'analyse' if session is None else 'analyse_%d' % int(session) 
```